recipes/sphinx_rtd_theme/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.9" %}
+{% set sha256 = "273846f8aacac32bf9542365a593b495b68d8035c2e382c9ccedcac387c9a0a1" %}
+
+package:
+  name: sphinx_rtd_theme
+  version: {{ version }}
+
+source:
+  fn: sphinx_rtd_theme-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - sphinx_rtd_theme
+
+about:
+  home: https://github.com/snide/sphinx_rtd_theme/
+  license: MIT
+  summary: 'ReadTheDocs.org theme for Sphinx, 2013 version.'
+
+  description: |
+    This is a mobile-friendly sphinx theme made for readthedocs.org. It’s currently in
+    development there and includes some rtd variable checks that can be ignored if you’re
```

Comment:
Oh those are hard to find :smile: 

I am trying to fix the linter so we don't suffer from this.